Directory.Build.props

```diff
@@ -14,6 +14,12 @@
     <AnalysisMode>Recommended</AnalysisMode>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
+    <BeforeTargetFrameworkInferenceTargets>$(RepositoryEngineeringDir)BeforeTargetFrameworkInference.targets</BeforeTargetFrameworkInferenceTargets>
   </PropertyGroup>
 
+  <ItemGroup>
+    <SupportedPlatform Condition="'$(TargetPlatformIdentifier)' != '' and
+                                  '$(TargetPlatformIdentifier)' != 'windows'" Include="unix" />
+  </ItemGroup>
```

Comment:
I wonder if you could move this into BeforeTargetFrameworkInference.targets and unconditionally include `$(_targetPlatformIdentifier)`. I.e. when targeting net8.0-unix, `unix` would then get added automatically. Same would be true for any other potential platform.